packages/admin-ui/src/data-view/components/data-view-status.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+
+import { CollectionError } from '../../components/Illustrations'
+
+import { useMessageFormatter } from '../../i18n'
+import { messages } from '../data-view.i18n'
+import { useDataViewContext } from '../context'
+import * as styles from './data-view.styles'
+import type { StatusAction } from '../data-view.state'
+import { Button } from '../../button'
+import { Stack } from '../../stack'
+import { Center } from '../../center'
+import { Box } from '../../box'
+
+export function DataViewStatus() {
+  const { status, statusObject } = useDataViewContext()
+
+  const formatMessage = useMessageFormatter(messages)
+
+  if (!status) return null
+
+  if (status === 'loading' || status === 'ready') return null
```

Comment:
Do you think it would decrease the readability if you merge those conditions in just one `if block`?

Reply:
I think it won't, I'll update it